pkg/image/util/manifest.go

```diff
@@ -0,0 +1,43 @@
+package util
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"github.com/docker/distribution/digest"
+
+	"github.com/openshift/origin/pkg/image/importer"
+)
+
+// GetImageManifestFromRegistry retrieves the image manifest from the registry using the basic
+// authentication using the image ID.
+func GetImageManifestFromRegistry(registry *url.URL, repositoryName, imageID, username, password string) ([]byte, error) {
```

Comment:
Can you name it like `GetImageManifestByIDFromRegistry()` since it doesn't support tags?